ipaserver/install/kra.py

```diff
@@ -85,23 +67,15 @@ def install(api, replica_config, options, custodia):
         if not replica_config.setup_kra:
             return
         krafile = os.path.join(replica_config.dir, 'kracert.p12')
-        if options.promote:
-            with ipautil.private_ccache():
-                ccache = os.environ['KRB5CCNAME']
-                kinit_keytab(
-                    'host/{env.host}@{env.realm}'.format(env=api.env),
-                    paths.KRB5_KEYTAB,
-                    ccache)
-                custodia.get_kra_keys(
-                    krafile,
-                    replica_config.dirman_password)
-        else:
-            cafile = os.path.join(replica_config.dir, 'cacert.p12')
-            if not os.path.isfile(cafile):
-                raise RuntimeError(
-                    "Unable to clone KRA."
-                    "  cacert.p12 file not found in replica file")
-            shutil.copy(cafile, krafile)
```

Comment:
This is the part where replica_config is not None - therefore replica installation. shutil.copy(cafile, krafile) is executed in the else tree of "if options.promote:", therefore only for DL0. It will therefore not be used at all for DL1.